.github/workflows/test-e2e.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           done
       - name: Checkout template-operator
         uses: actions/checkout@v3
-        if: ${{ matrix.e2e-test == 'module-status-propagation' || matrix.e2e-test == 'module-without-default-cr' || matrix.e2e-test == 'non-blocking-deletion'}}
+        if: ${{ matrix.e2e-test == 'module-status-propagation' || 'module-without-default-cr' || 'non-blocking-deletion' }}
```

Comment:
this if condition is not working, you need that matrix.e2e-test == 'module-without-default-cr', otherwise, it just means `if 'module-without-default-cr'` which will always `true`, so basically means this step will run in each e2e test

Reply:
Ups, leftover from merge conflict